### ISSUE TITLE
Instructors should always be able to respond to complaints and more feedback requests

### ISSUE THREAD

**Issue by krusche** [ls1intum/Artemis#2027]
at the moment the logic in `assessment-layout.component.ts` prevents this:

```
    /**
     * For team exercises, the team tutor is the assessor and handles both complaints and feedback requests himself
     * For individual exercises, complaints are handled by a secondary reviewer and feedback requests by the assessor himself
     */
    get isAllowedToRespond(): boolean {
        if (this.complaint!.team) {
            return this.isAssessor;
        } else {
            return this.complaint!.complaintType === ComplaintType.COMPLAINT ? !this.isAssessor : this.isAssessor;
        }
    }
```

However, instructors should always be able to work on complaints

**Comment by krusche** [comment]
Same for more feedback requests

**Comment by Strohgelaender** [comment]
Fixed by #3019
